README.md

```diff
@@ -1,40 +1,46 @@
 # Introduction
 
-This repository has two  examples that show how your existing Apache Spark applications can be deployed and configured seamlessly using CDAP. 
+This guide will demonstrate how to manage and run your existing Apache Spark application in CDAP. This repository has two examples and a step-by-step instructions in acheiving this. 
 
-This capability allows users of CDAP to run their existing Spark application in CDAP without making any modification to the Spark code.
+Users will be able to run their existing Spark application in CDAP without making any modifications to the Spark code.
 
-This repository contains two pre-built example spark applications.
+Sample spark examples used in this guide:
 
   - word-count-java: contains word count example written in Java 
-  - sparkpi-scala: contains sparkpi example written in Scala [how-to](SCALA-SPARK.md)
+  - sparkpi-scala: contains sparkpi example written in Scala 
 
-# How To 
+The following instructions are for deploying the word-count-java Spark application. For deploying Scala Spark app, follow the instructions [here](SCALA-SPARK.md). 
```

Comment:
nit: `For deploying sparkpi-scala application,`

to be consistent with `word-count-java`